src/components/ReferralButton.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import {
+  Button, Dropdown, Card, Input, Space, Tooltip,
+} from 'antd';
+import { ShareAltOutlined, DownOutlined } from '@ant-design/icons';
+import { Auth } from 'aws-amplify';
+import endUserMessages from '../utils/endUserMessages';
+import pushNotificationMessage from '../utils/pushNotificationMessage';
+
+const { TextArea } = Input;
+
+const initialMessage = 'Hi,\n\nCheck out Cellscope from Biomage. It will make your single-cell analysis easier.';
+
+// Valid email regex based on RFC2822 - https://regexr.com/2rhq7
+const emailRegex = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
```

Comment:
Why don't you validate the email from `utils/validateInputs.js` like you have added VALIDATE_EMAIL at the end of the changes in this PR?

Reply:
hahah yea, added the email check to `validateInputs`

Reply:
hahah yea, it's added now